apps/backend/src/mutations/ProposalMutations.ts

```diff
@@ -752,4 +757,34 @@ export default class ProposalMutations {
 
     return submitted;
   }
+
+  @Authorized([Roles.USER_OFFICER])
+  async sendEventEmail(
+    agent: UserWithRole | null,
+    args: SendProposalEventEmailArgs
+  ) {
+    if (args.event !== Event.PROPOSAL_SUBMITTED) {
+      return rejection('Proposal send email event requested not supported');
+    }
+
+    const proposal = await this.proposalDataSource.getProposalById(
+      args.proposalId
+    );
+
+    if (!proposal) {
+      return rejection('Proposal not found', { args });
+    }
+
+    if (!proposal.submitted) {
+      return rejection('Proposal has not been submitted', { args });
+    }
+
+    return this.emailHandler({
+      type: Event.PROPOSAL_SUBMITTED,

```

Comment:
Shouldn't we use `type: args.event` here instead?

Reply:
I have fixed this.